### DISCUSSION TITLE
[FEATURE-REQUEST] ¿Cómo puede haber trazabilidad de un correo enviado desde cotalker y lo responda el destinatario?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#175] [3. Ideas & Feature Requests] [answered]
Hola el título creo que no se entiende pero lo explicaré con pasos.

1.- un usuario no cotalker utiliza un survey extern y envía una solicitud
2.- Llega un correo a un usuario cotalker como notificación + el link del canal de la task que se creó.
3.- El usuario cotalker ingresa al link con su cuenta y ve el formulario.
4.- El usuario cotalker levanta un formulario de validación y entre las respuestas indica que falta información adicional y lo envía.
5.- Cotalker envía un correo al usuario (mediante un nwr answers), con el campo información adicional.
6.- Usuario no cotalker responde el correo.

Ahora mi pregunta es: Cómo se puede, si es que se puede, lo que responde el correo el usuario no cotalker, llegue a cotalker al canal respectivo?

Creo, por lo que tengo conocimientos, no se puede, ahora si se puede me gustaría saber cómo.

Gracias

**Comment by rafaelwalter** [comment, marked as answer]
@NicolasPLG no se puede implementar esta integracion con Cotalker.

Puedes hacer una integracion con mail ocupando [Zapier](https://zapier.com/) que se conecte con los bots de Cotalker.
Por otro lado, puedes ver como se implementó en el demo de Validaciones una solución similar a la que necesitas.

**Reply by NicolasPLG** [reply]
Hola @rafaelwalter perfecto gracias por la información.
No tengo credenciales sobre lo uno (Zapier o no sé cuáles usuarios tienen las credenciales) y sobre el demo de Validaciones, me puedes dar las credenciales por interno para revisar el tema?

Gracias

**Reply by rafaelwalter** [reply]
@NicolasPLG Zapier es 100% de responsabilidad de los implementadores, Cotalker no entrega soporte o cuentas de usuario. Además es una herramienta paga y que el costo deberia ser pasado al cliente.

Sobre el demo favor investigar con el equipo comercial que preparó el demo, el equipo de ingenieria no tiene acceso a esa información.

**Reply by NicolasPLG** [reply]
perfecto gracias @rafaelwalter por la información.

Atte.
